Source/Csla.AspNetCore/Blazor/ApplicationContextManagerBlazor.cs

```diff
@@ -160,8 +166,10 @@ public IContextDictionary GetLocalContext()
     /// Sets the local context.
     /// </summary>
     /// <param name="localContext">Local context.</param>
+    /// <exception cref="InvalidOperationException"><see cref="ApplicationContext"/> is <see langword="null"/>.</exception>
     public void SetLocalContext(IContextDictionary localContext)
     {
+      ThrowIfApplicationContextIsNull();
```

Comment:
As before, null should be a valid value here.

Reply:
It's value. But `ApplicationContext` can be `null` and instead of an `NullReferenceException` we are now throwing an `InvalidOperationException`. Imho that's the more correct exception here.